devops/roles/postgresql-restore/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+postgresql_restore_dir: /tmp/postgresql-restore
+postgresql_port: 5432
+postgresql_cluster_version: 9.5
+postgresql_cluster_name: main
+postgresql_user: postgres
+postgresql_restore_azure_container_name: {{ azure_container }}
+postgresql_restore_azure_storage_account_name: {{ azure_account_name }}
+postgresql_restore_azure_storage_access_key: {{ azure_storage_secret }}
+postgresql_restore_gzip_file_name: {{ backup_file_name }}
```

Comment:
I see that the backup file name from the backup scripts is configured to be `postgresql_backup_{{ lookup('pipe', 'date +%Z-%Y-%m-%d-%H-%M-%S') }}.sql.gz`. Are we using the same filename in the restoration configuration? Where are we overriding this configuration?